akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -23,6 +23,10 @@ trait ExceptionHandler extends ExceptionHandler.PF {
 
 object ExceptionHandler {
   type PF = PartialFunction[Throwable, Route]
+  private[http] lazy val errorMessageTemplate: String = {
```

Comment:
Sorry, I meant "mention ExceptionHandler's informative message in docs" :smiley: So basically document your changes in the docs.